jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/util/GenericErrorSummaryCountCell.java

```diff
@@ -104,8 +125,10 @@ private native void initTooltips(String dataAttrVal,
                 .tooltip()
                 .on("click", function () {
                     var processInstId = $wnd.jQuery(this).attr(procIdAttrName);
-                    thisCellRef.@org.jbpm.workbench.pr.client.editors.instance.list.ProcessInstanceSummaryErrorCountCell::openErrorView(Ljava/lang/String;)(processInstId);
+                    thisCellRef.@org.jbpm.workbench.common.client.util.GenericErrorSummaryCountCell::openErrorView(Ljava/lang/String;)(processInstId);
```

Comment:
I don't know much about JS and this looks to me as a purely cosmetic comment, but for readability reasons it would be good to change the variable name in the last bracket from ```(processInstId)``` to just ```(id)```.

Reply:
Done